src/System.Globalization/tests/StringInfo/StringInfoCtor.cs

```diff
@@ -9,30 +9,34 @@ namespace System.Globalization.Tests
 {
     public class StringInfoCtorTests
     {
-        private const int MinStringLength = 8;
-        private const int MaxStringLength = 256;
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
-
         [Fact]
         public void Ctor_Empty()
         {
             StringInfo stringInfo = new StringInfo();
             Assert.Equal(string.Empty, stringInfo.String);
+            Assert.Equal(0, stringInfo.LengthInTextElements);
         }
 
         public static IEnumerable<object[]> Ctor_String_TestData()
         {
-            yield return new object[] { s_randomDataGenerator.GetString(-55, false, MinStringLength, MaxStringLength) };
```

Comment:
It looks like this was generating a 244 character string.  There doesn't appear to be anything as long in what you've replaced this with...?